### PR TITLE
Fix numpy

### PR DESCRIPTION
**Pull request by oew1v07**
This adds the np.load function as an input, this will need to be merged prior to #207 as it depends on it being fixed.

### CODE REVIEW

File: integration_test/packages/run_numpy.py [hunk 1]
```diff
@@ -62,6 +62,10 @@ def genfromtxt(self):
         np.genfromtxt(file_name, delimiter=",",
                       missing_values="", filling_values=-1)
 
+    def load(self):
```
Comment (robintw) [inline integration_test/packages/run_numpy.py]: Could you add a very brief docstring here? (Like the other functions in this module)

**Comment by robintw** [comment]
Looks great - thank you. I've just added one simple comment on a docstring I'd like you to add.

 I see Travis has passed, I'll just wait for AppVeyor to pass too, and then merge this.

(Once this is merged, just pull from master in to your other PR and that should work too)

**Comment by oew1v07** [comment]
Right added that doc string. Just waiting for travis and appveyor.

**Comment by robintw** [comment]
This is passing for all except 3.5 on AppVeyor, and that seems to be because it timed out due to some weird conda download slowdowns. Merging anyway, as it's only a small change.

**Pull request merged**